src/main/java/uk/gov/hmcts/reform/civil/service/search/TrialReadyNotificationSearchService.java

```diff
@@ -33,6 +33,7 @@ public Query query(int startIndex) {
                                                                         .atTime(LocalTime.MIN)
                                                                         .plusWeeks(6).toString()))
                             .must(beState(PREPARE_FOR_HEARING_CONDUCT_HEARING)))
+                            .mustNot(matchQuery("data.allocatedTrack", "SMALL_CLAIM"))
```

Comment:
Can be replaced with the enum constant AllocatedTrack.SMALL_CLAIM.toString() value. It would avoid having a hard-coded string.